.env

```diff
@@ -1 +1,2 @@
 KAFKA_CONFLUENT_VERSION=6.1.0
+GOOGLE_CREDENTIALS_FILE_PATH=/Users/yatharth/Downloads/mind-course-firebase-adminsdk-8nyar-9d05087b3d.json
```

Comment:
do you want the google cred file path hardcoded?

Reply:
Thanks will remove.